src/plugin/context/mod.rs

```diff
@@ -22,13 +24,30 @@ use crate::prelude::{CollisionGroups, RapierRigidBodyHandle};
 use rapier::control::CharacterAutostep;
 use rapier::geometry::DefaultBroadPhase;
 
+/// Marker component for to access the default [`RapierContext`].
+///
+/// This is used by [`systemparams::DefaultRapierContextAccess`] and other default accesses
+/// to help with getting a reference to the correct RapierContext.
+///
+/// If you're making a library, you might be interested in [`RapierContextEntityLink`]
+/// and leverage a [`Query<&RapierContext>`] to find the correct [`RapierContext`] of an entity.
+#[derive(Component, Reflect, Debug, Clone, Copy)]
+pub struct DefaultRapierContext;
+
+/// This is a component applied to any entity containing a rapier handle component.
+/// The inner Entity referred to has the component [`RapierContext`] responsible for handling
+/// its rapier data.
+#[derive(Component, Reflect, Debug, Clone, Copy, PartialEq, Eq)]
+pub struct RapierContextEntityLink(pub Entity);
```

Comment:
I like this change. I think it will make it more obvious when a world is no longer valid, since it uses the typical ECS architecture. It also makes it easier to add/remove components from a world if you would chose to do that.